keps/sig-network/2004-topology-aware-subsetting/README.md

```diff
@@ -565,6 +583,165 @@ label. Each approach may group the endpoints in slightly different
 combinations of EndpointSlices, but it will still be just as easy to list all
 endpoints for a Service.
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: TopologyAwareSubsetting
+    - Components depending on the feature gate: kube-controller-manager
+  - [x] Other
+    - Describe the mechanism:
+      `endpointslice.kubernetes.io/subsetting` annotation on Service.
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+      No
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node?
+      No
+  - [x] Other
+    - Describe the mechanism:
+      Controller flag to change inferred value when annotation is not set.
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+      No
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node?
+      No
+
+* **Does enabling the feature change any default behavior?**
+  In most cases, this feature will be intentionally enabled on specific Services
```

Comment:
Agree with that. I'll update the KEP to target default on by GA.

Reply:
See my above comment - it might be valuable to position default-on as a different KEP with a different gate, unless we are willing to push GA out by a few releases, possibly.

Reply:
Good call, added a note describing that any change to default behavior would be done in a future KEP.